### PR TITLE
IBM Resilient safe access dictionaries

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-15849)

## Description
A safe access to dictionaries was added, due to unknown keys in customers.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IBM Resilient Systems - Dev
+- Improved implementation.
```
Comment (yuvalbenshalom) [inline Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md:4]:
```suggestion
- fixed an issue where command **rs-get-inciden** failed to run.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comment.

File: Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IBM Resilient Systems - Dev
+- fixed an issue where command **rs-get-inciden** failed to run.
```
Comment (rshunim) [inline Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md:4]:
```suggestion
- Fixed an issue where command **rs-get-inciden** failed to run.
```

File: Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IBM Resilient Systems - Dev
+- Fixed an issue where command **rs-get-inciden** failed to run.
```
Comment (rshunim) [inline Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md:4]:
```suggestion
- Fixed an issue where command **rs-get-incident** failed to run.
```

File: Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IBM Resilient Systems - Dev
+- Fixed an issue where command **rs-get-incident** failed to run.
```
Comment (rshunim) [inline Packs/IBMResilientSystems/ReleaseNotes/1_1_3.md:4]:
```suggestion
- Fixed an issue where ***rs-get-incident*** command failed to run.
```

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**